unlock-app/src/hooks/useMultipleRecipient.ts

```diff
@@ -89,15 +91,43 @@ export const useMultipleRecipient = (
 
   const getAddressAndValidation = async (recipient: string) => {
     const address = await getAddressForName(recipient)
+    const existingExpiration = await web3Service.getKeyExpirationByLockForOwner(
```

Comment:
Let's use `getHasValidKey` because it's compatible with all versions :)

Reply:
This way you don't have to deal with the expiration logic , which simplifies things!